README.md

```diff
@@ -42,10 +42,10 @@ Running on Kubernetes? See [runtime/kubernetes-1ppc](./runtime/kubernetes-1ppc)
 The default container configuration is as follows:
 
  - stackstorm (st2 + st2web + st2mistral)
- - mongo
- - rabbitmq
- - postgres
- - redis
+ - st2mongo
+ - st2rabbitmq
+ - st2postgres
+ - st2redis
```

Comment:
Aren't these changes confusing? We should just list the name of external libs/components, not the container names.